src/applications/accreditation/21a/pages/helpers/formatReviewDate.js

```diff
@@ -24,9 +37,9 @@ export const formatReviewDate = (dateString, monthYear = false) => {
     }
 
     const formattedMonth = monthNames[parseInt(month, 10) - 1];
-    const formattedDay = parseInt(day, 10).toString();
+    const formattedDay = day ? parseInt(day, 10).toString() : '';
```

Comment:
Is `parseInt().toString()` needed here? `day` would go from a string to an integer, and then back to a string, unless I'm misinterpreting this.